### PR TITLE
Responsive contact

### PR DESCRIPTION
**Pull request by MangeFre**
[Ticket](https://trello.com/c/bbYvtfBO)

### CODE REVIEW

File: scss/main.scss [hunk 1]
```diff
@@ -356,13 +356,23 @@ h1,h2,h3 {
...
+  @media(max-width: 360px){
+    padding: 48px 24px;
+  }
+
   h1 {
     color: $white;
-    margin-bottom: 2rem;
     font-weight: 700;
+    @media(max-width: 651px){
+      font-size: 32px;
```
Comment (jrapala) [inline scss/main.scss:374]: Let's please use `rem` instead of `px`.
This will allow users to scale fonts up if they need to.

**Review by jrapala — CHANGES_REQUESTED** [review]

**Review by jrapala — COMMENTED** [review]
Can you also resolve the merge conflict? Thanks!

File: scss/main.scss (hunk 1 above)
Comment (MangeFre) [reply scss/main.scss]: Thanks for the tip! It should be fixed now.

**Comment by MangeFre** [comment]
Merge conflict + Comments should be resolved now.

**Review by jrapala — APPROVED** [review]

**Pull request merged**